### PR TITLE
Change to always restart

### PR DESCRIPTION
**Pull request by cheran-senthil**
### Description
Change to always restart with autoheal

### CODE REVIEW

**Review by dmanjunath — APPROVED** [review]
i see what happened here. discovery had all this stuff but content never did.

**Comment by cheran-senthil** [comment]
Yes, exactly

**Review by SidSethi — APPROVED** [review]
nice find!

File: common-services.yml [hunk 1]
```diff
@@ -21,9 +22,10 @@ services:
 
   base-cache:
     image: redis:5.0.5
+    restart: always
```
Comment (dmanjunath) [inline common-services.yml:25]: lets keep this in content node only. this is due to a content node limitation. if redis restarts the server also needs to restart

**Review by dmanjunath — CHANGES_REQUESTED** [review]

**Pull request merged**